docs/application-connector/03-06-api-registration.md

```diff
@@ -15,3 +15,11 @@ You can pass the API specification in two ways:
 
 For the OpenAPI format, both methods are supported.
 You can register OData APIs only with `SpecificationUrl`.
+
+You can specify additional headers and query parameters with requests to:
+- make a call to the target URL
+- fetch the API specification
+- fetch an OAuth access token
```

Comment:
I think I wouldn't mention the OAuth call this way. The calls to the target URL and for the API specification are explicit. On the other the OAuth call is implicit, user doesn't even have to remember that such call is performed. It's done under the hood. I'd put here something like `- authenticate with custom OAuth method`